test/event_test.py

```diff
@@ -28,6 +28,70 @@
 )
 
 
+names_and_events = (
+    ('NoEvent', pygame.NOEVENT),
+    ('ActiveEvent', pygame.ACTIVEEVENT),
+    ('KeyDown', pygame.KEYDOWN),
+    ('KeyUp', pygame.KEYUP),
+    ('MouseMotion', pygame.MOUSEMOTION),
+    ('MouseButtonDown', pygame.MOUSEBUTTONDOWN),
+    ('MouseButtonUp', pygame.MOUSEBUTTONUP),
+    ('JoyAxisMotion', pygame.JOYAXISMOTION),
+    ('JoyBallMotion', pygame.JOYBALLMOTION),
+    ('JoyHatMotion', pygame.JOYHATMOTION),
+    ('JoyButtonDown', pygame.JOYBUTTONDOWN),
+    ('JoyButtonUp', pygame.JOYBUTTONUP),
+    ('VideoResize', pygame.VIDEORESIZE),
+    ('VideoExpose', pygame.VIDEOEXPOSE),
+    ('Quit', pygame.QUIT),
+    ('SysWMEvent', pygame.SYSWMEVENT),
+
+    ('UserEvent', pygame.USEREVENT),
+    ('UserEvent', pygame.USEREVENT + 1),
+    ('UserEvent', pygame.NUMEVENTS - 1),
+
+    ('Unknown', 0xFFFF),
+)
+
+# Add in any SDL 2 specific events.
+if pygame.get_sdl_version()[0] >= 2:
+        names_and_events += (
```

Comment:
This indentation seems to differ from the rest of the file

Reply:
Thanks for the comment. I've made the requested change.